src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -951,10 +955,10 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
                                 controllerHandler.sendData(node.encapsulate(userCodeCommandClass.getUserCode(code), 0));
                                 pendingCfg.put(configurationParameter.getKey(), valueObject);
                             } else {
-                                logger.error("Value format error processing user code {}", valueObject);
+                                logger.debug("Value format error processing user code {}", valueObject);
```

Comment:
Probably this should be WARN - this is quite a serious error.

Reply:
Changed to warn.